### PR TITLE
Added missing newline in error message

### PR DESCRIPTION
**Pull request by Teemperor**

### CODE REVIEW

**Comment by Teemperor** [comment]
We will sadly loose this hypnotizing piece of art with this patch: https://teemperor.de/pub/rootio_errors.png

**Comment by vgvassilev** [comment]
@Teemperor, could you address the clang-format complaint?

**Comment by amadio** [comment]
@Teemperor ping. Please clang-format so we can merge. The change seems sensible afterall.

File: core/clingutils/src/TClingUtils.cxx [hunk 1]
```diff
@@ -2887,7 +2887,7 @@ clang::QualType ROOT::TMetaUtils::AddDefaultParameters(clang::QualType instanceT
                if (ArgType.getArgument().isNull()
                    || ArgType.getArgument().getKind() != clang::TemplateArgument::Type) {
                   ROOT::TMetaUtils::Error("ROOT::TMetaUtils::AddDefaultParameters",
-                                          "Template parameter substitution failed for %s around %s",
+                                          "Template parameter substitution failed for %s around %s\n",
                                           instanceType.getAsString().c_str(),
                                           SubTy.getAsString().c_str()
                                           );
```
Comment (amadio) [inline core/clingutils/src/TClingUtils.cxx]: You need to join lines 2891-2893 for please clang-format...

Comment (Teemperor) [inline core/clingutils/src/TClingUtils.cxx]: Sorry, forgot to add the file after clang-format...

**Pull request merged**